cnxepub/collation.py

```diff
@@ -33,7 +33,8 @@ def easybake(ruleset, in_html, out_html):
     with respective read and write ability.
 
     """
-    html = etree.parse(in_html)
+    html_parser = etree.HTMLParser(encoding="utf-8")
```

Comment:
I missed the discussion, what was the reason for using the HTML parser? I didn't see that part explained in the PR description